sdk/go/synthetics/pulumi-plugin.json

```diff
@@ -1,4 +1,5 @@
 {
   "resource": true,
-  "name": "synthetics"
+  "name": "synthetics",
+  "server": "https://github.com/Displayr/pulumi-signalfx-synthetics/releases/download/v0.0.11/"
```

Comment:
Ditto

Reply:
yep, the pipeline re-generate everything under /sdk folder.

I will re-run `make build-sdk` locally.